Source/Core/Core/PowerPC/JitArm64/JitArm64_LoadStoreFloating.cpp

```diff
@@ -71,17 +71,14 @@ void JitArm64::lfXX(UGeckoInstruction inst)
 	u32 imm_addr = 0;
 	bool is_immediate = false;
 
-	bool only_lower = !!(flags & BackPatchInfo::FLAG_SIZE_F64);
+	RegType type = !!(flags & BackPatchInfo::FLAG_SIZE_F64) ? REG_LOWER_PAIR : REG_DUP;
 
-	fpr.BindToRegister(inst.FD, false, only_lower);
+	fpr.BindToRegister(inst.FD, false, type);
 
-	ARM64Reg VD = fpr.R(inst.FD, only_lower);
+	ARM64Reg VD = fpr.R(inst.FD, type);
 	ARM64Reg addr_reg = W0;
 
-	if (!fpr.IsLower(inst.FD))
-		only_lower = false;
-
-	if (only_lower)
+	if (type == REG_LOWER_PAIR)
 		flags |= BackPatchInfo::FLAG_ONLY_LOWER;
```

Comment:
Is this flag needed any more at all? Sounds like FLAG_SIZE_F64 and FLAG_ONLY_LOWER are the same now.